src/modules/auth/actions/load-account-data.js

```diff
@@ -11,4 +17,12 @@ export const loadAccountData = account => (dispatch) => {
   dispatch(updateLoginAccount(account))
   dispatch(loadAccountPositions())
   dispatch(updateAssets())
+  dispatch(checkAccountAllowance((err, allowance) => {
+    if (err) return console.log(err)
```

Comment:
Suggest using the usual callback-with-default-`logError` instead of hardcoding in `console.log` here.